D4Companion.Services/ScreenProcessHandler.cs

```diff
@@ -130,867 +113,514 @@ private void initImageList()
 
-            await Task.Run(() =>
+            _eventAggregator.GetEvent<TooltipDataReadyEvent>().Publish(new TooltipDataReadyEventParams
             {
-                try
-                {
-                    var watch = System.Diagnostics.Stopwatch.StartNew();
-
-                    // Clear previous tooltip
-                    _currentTooltip = new ItemTooltipDescriptor();
-
-                    if (_currentScreen.Height < 50)
-                    {
-                        _logger.LogWarning($"{MethodBase.GetCurrentMethod()?.Name}: Diablo IV window is probably minimized.");
-                        return;
-                    }
-
-                    bool result = FindTooltips();
-                    if (result) 
-                    {
-                        result = FindItemTypes();
-                    }
-                    if (result)
-                    {
-                        result = FindItemAffixLocations();
-                    }
-                    if (result)
-                    {
-                        // Result does not matter, always continue.
-                        FindItemAffixes();
-
-                        result = FindItemAspectLocations();
-                    }
-                    if (result)
-                    {
-                        // Result does not matter, always continue.
-                        FindItemAspects();
-                    }
-
-                    _logger.LogDebug($"{MethodBase.GetCurrentMethod()?.Name}: Tooltip data ready:");
-                    _logger.LogDebug($"{MethodBase.GetCurrentMethod()?.Name}: Item type: {_currentTooltip.ItemType}");
-                    _logger.LogDebug($"{MethodBase.GetCurrentMethod()?.Name}: Item affixes: {_currentTooltip.ItemAffixes.Count}");
-                    _logger.LogDebug($"{MethodBase.GetCurrentMethod()?.Name}: Item aspect: {!_currentTooltip.ItemAspect.IsEmpty}");
-
-                    _eventAggregator.GetEvent<TooltipDataReadyEvent>().Publish(new TooltipDataReadyEventParams
-                    {
-                        Tooltip = _currentTooltip
-                    });
+                Tooltip = _currentTooltip
+            });
+        }
 
-                    watch.Stop();
-                    var elapsedMs = watch.ElapsedMilliseconds;
-                    _logger.LogDebug($"{MethodBase.GetCurrentMethod()?.Name}: Total Elapsed time: {elapsedMs}");
+        private static void DrawRect<TColor>(Image<TColor, byte> target, Rectangle rect) where TColor : struct, IColor
+        {
+            CvInvoke.Rectangle(target, rect, new MCvScalar(0, 0, 255));
+        }
 
-                }
-                catch (Exception ex)
-                {
-                    _logger.LogError(ex, MethodBase.GetCurrentMethod()?.Name);
-                }
-            });
+        private static void FillRect<TColor>(Image<TColor, byte> target, Rectangle rect, MCvScalar color) where TColor : struct, IColor
```

Comment:
Why create a new one-line static function for an existing static function?
Would prefer to call `CvInvoke.Rectangle` directly.